pkg/helper/hasher.go

```diff
@@ -0,0 +1,5 @@
+package helper
+
+func HashKey(key []byte, buckets uint32) uint32 {
```

Comment:
Why is this function in the helper package?

Reply:
We will have a separate package for hash functions, with multiple implementations for comparison purposes. This has a related issue in #2.